pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -804,6 +877,15 @@ func loadConfigFile(path string, forceV2 bool) (*parsedConfig, error) {
 		res.shortNameMode = types.ShortNameModeInvalid
 	}
 
+	// Valid wildcarded prefixes must be in the format: *.example.com
+	for i := range res.partialV2.Registries {
+		prefix := res.partialV2.Registries[i].Prefix
+		if prefix[:2] == "*." && strings.ContainsAny(prefix, "/@:") {
```

Comment:
For the future: This should move into `postProcessRegistries`.

Reply:
noted.